sim/rogue/items_sets.go

```diff
@@ -1,20 +1,71 @@
 package rogue
 
-import "github.com/wowsims/sod/sim/core"
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/stats"
+)
 
 ///////////////////////////////////////////////////////////////////////////
 //                            SoD Phase 3 Item Sets
 ///////////////////////////////////////////////////////////////////////////
 
-// TODO: New Set Bonuses
 var ItemSetBloodCorruptedLeathers = core.NewItemSet(core.ItemSet{
 	Name: "Blood Corrupted Leathers",
 	Bonuses: map[int32]core.ApplyEffect{
 		2: func(agent core.Agent) {
-			// c := agent.GetCharacter()
+			c := agent.GetCharacter()
+			c.AddStat(stats.MeleeHit, 1)
```

Comment:
Same comment about this set bonus. I think we should consider leaving this inactive until we can properly implement the proc damage effect